spec/system/visit_service_new_spec.rb

```diff
@@ -18,6 +18,17 @@
 
       expect(current_path).to eql admin_path
     end
+
+    it 'when entity id has only leading and trailing spaces' do
```

Comment:
Same comment here about testing - can we check the leading and trailing whitespaces have been removed?